### ISSUE TITLE
When my runtime environment is a Chinese encoding environment, the list of error messages returned is garbled

### ISSUE THREAD

**Issue by GodMeowIceSun** [nahsra/antisamy#456]
### input

![image](https://github.com/nahsra/antisamy/assets/53115201/d2a22a0a-4c9b-4590-9409-3c9b6cac9cc6)

### critical code

![image](https://github.com/nahsra/antisamy/assets/53115201/cc629bdd-ebdc-4539-b6ea-c094e857f361)

### result

![image](https://github.com/nahsra/antisamy/assets/53115201/e62fd8f0-a056-43ec-a7fb-a37e1701609b)

### individual opinion

There is a problem with different character sets, so I think the output content of the default character set should be provided.

**Comment by GodMeowIceSun** [comment]
This issue occurs when I use dependencies in Maven, but it no longer exists after I compile locally

**Comment by GodMeowIceSun** [comment]
No, it's not that there's no problem with local compilation, but rather that getErrorMessages has been called multiple times

**Comment by GodMeowIceSun** [comment]
After applying changes and compiling locally, There's no garbled code left.

### result

![image](https://github.com/nahsra/antisamy/assets/53115201/3a6e643e-d6db-490b-957a-0d3af17c9c1a)

**Comment by davewichers** [comment]
Per @kwwall - I suppose the correct thing to do to address this is for someone to correct all the AntiSamy properties files and XML files as per the [Oracle JDK 9 document](https://docs.oracle.com/javase/9/intl/internationalization-enhancements-jdk-9.htm#JSINT-GUID-974CF488-23E8-4963-A322-82006A7A14C7) that [@GodMeowIceSun](https://github.com/GodMeowIceSun) referenced, and use UTF-8 instead of ISO-8859-1. (And if that doesn't work, use a system property like I suggested earlier.)

**Comment by davewichers** [comment]
@spassarop - Do you think you'd be able to work on this at some point? It's been dangling out there for a while.

**Comment by spassarop** [comment]
@GodMeowIceSun I made changes on PR #569. Could you test that project branch on your environment?

You can reproduce with your own code but you could also run this on a JUnit test case (in `AntiSamyTest.java`) to print all messages in the bundle:
```java 
    Enumeration<String> keys = messages.getKeys();
    while (keys.hasMoreElements()) {
      String key = keys.nextElement();
      String value = messages.getString(key);
      System.out.println(key + " = " + value);
    }
```
